external/buildscripts/build_win_no_cygwin.pl

```diff
@@ -39,7 +39,7 @@
 my $buildDeps = "";
 my $stevedoreBuildDeps=1;
 
-if($ENV{YAMATO_PROJECT_ID} || ($ENV{USERNAME} == "bokken"))
+if($ENV{YAMATO_PROJECT_ID} || ($ENV{USERNAME} eq "bokken"))
```

Comment:
I also think if you want to see if YAMAO_PROJECT_ID is set then you'll want to do something like: `if (defined $ENV{'YAMATO_PROJECT_ID'} || ...`